CHANGELOG.md

```diff
@@ -2,7 +2,23 @@
 
 ## unreleased
 
+## 3.2.0 RC0 / 2025-01-29
+
+* [FEATURE] remote/otlp: add feature flag `otlp-deltatocumulative` to support conversion from delta to cumulative. #15165
 * [ENHANCEMENT] promtool: Support linting of scrape interval, through lint option `too-long-scrape-interval`. #15719
+* [ENHANCEMENT] openstack SD: discover Octavia loadbalancers. #15539
+* [ENHANCEMENT] scrape: Add metadata for automatic metrics for WAL. #15837
+* [ENHANCEMENT] promtool: add --ignore-unknown-fields option. #15706I
+* [ENHANCEMENT] promtool: Support linting of scrape interval. #15719
```

Comment:
Could we keep all the promtool changes together?

Reply:
Yes good point, thank you!